### ISSUE TITLE
NaN value for val/loss and val/vote_loss

### ISSUE THREAD

**Issue by Divyam10** [zju3dv/clean-pvnet#235]
Hey @pengsida, 

I created a custom dataset using BlenderProc in the format mentioned. The model starts training and the losses decrease as well, however val/loss and val/vote_loss remain NaN. 

I have also used ObjectDataSetTools to create the dataset and I do not get NaN values for that dataset. 

I have replicated the exact format of ObjectDataSetTools to generat synthetic data using BlenderProc. I have visualised the gt_poses as well and they look fine. 

Could you help me with this? 

![Screenshot from 2023-01-04 10-38-47](https://user-images.githubusercontent.com/29382463/210488931-c4345c65-9631-4768-ba67-2e34312504bd.png)
![Screenshot from 2023-01-04 10-45-02](https://user-images.githubusercontent.com/29382463/210488935-232e8e8c-90f1-4bb7-a80c-7208a9d94453.png)

**Comment by pengsida** [comment]
The validation metrics seem good. I think that you could ignore this problem. Maybe it is a bug caused by pytorch.
